go/kbtest/chat.go

```diff
@@ -491,6 +493,8 @@ func (m *ChatRemoteMock) PostRemote(ctx context.Context, arg chat1.PostRemoteArg
 	conv := m.world.GetConversationByID(arg.ConversationID)
 	ri := m.makeReaderInfo(conv.Metadata.ConversationID)
 	inserted := m.insertMsgAndSort(arg.ConversationID, arg.MessageBoxed)
+	m.world.T.Logf("PostRemote(convid:%v, msgid:%v, %v)",
```

Comment:
Do you want to keep this?

Reply:
I like test logs. Can get rid of it if you'd like

Reply:
Sure thats fine